src/components/Tool/Appeal/AppealDetails/AppealsMainPanel/AppealsMainPanelHeader.tsx

```diff
@@ -38,6 +38,12 @@ const StyledToggleButtonGroup = styled(ToggleButtonGroup)(({ theme }) => ({
   marginLeft: theme.spacing(1),
 }));
 
+const StyledBox = styled(Box)(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  marginRight: '12px',
```

Comment:
Should we use `theme.spacing()`?

```suggestion
  marginRight: theme.spacing(1.5),
```